.changeset/itchy-toys-march.md

```diff
@@ -0,0 +1,21 @@
+---
+'astro': major
+---
+
+The option `security.checkOrigin` is `true` by default
+
+### Changed: `security.checkOrigin` default value to `true`
+In Astro v4.0, the value of `security.checkOrigin` was `false`, and users needed to opt-in.
+
+Astro v5.0, the default value of `security.checkOrigin` is now `true`.
+#### What should I do?
+
+Update `astro.config.mjs` to _opt-out_:
+
+```diff
+export default defineConfig({
++  security: {
++    checkOrigin: false
++  }
+})
+```
```

Comment:
``````suggestion
If you had previously configured `security.checkOrigin: true`, you no longer need this set in your Astro config. This is now the default and it is safe to remove.

To disable this behavior and opt out of automatically checking that the “origin” header matches the URL sent by each request, you must explicitly set `security.checkOrigin: false`:

```diff
// astro.config.mjs
export default defineConfig({
+  security: {
+    checkOrigin: false
+  }
})
```
``````
This is one I had already mocked up in the upgrade guide, so we can use it!  :smile: